src/utils/file.utils.ts

```diff
@@ -0,0 +1,194 @@
+const DEFAULT_IMAGE_WIDTH = 2560;
+const DEFAULT_IMAGE_QUALITY = 0.9;
+const WORD_SIZE = 4;
```

Comment:
This should be passed as an option from the consumer. They might want higher quality or larger width
```suggestion
const DEFAULT_IMAGE_WIDTH = 2560;
const DEFAULT_IMAGE_QUALITY = 0.9;
const WORD_SIZE = 4;
```

Reply:
hey @yashim-deriv 
`DEFAULT_IMAGE_WIDTH` and `DEFAULT_IMAGE_QUALITY` we already passing as options in `compressImage` function in `options` parameter.
for `WORD_SIZE` i did the change and added arraySize parameter to `numToUint8Array` function